go/internal/feast/onlinestore/cassandraonlinestore.go

```diff
@@ -0,0 +1,392 @@
+package onlinestore
+
+import (
+	"context"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/feast-dev/feast/go/internal/feast/registry"
+	"github.com/feast-dev/feast/go/internal/feast/utils"
+	"github.com/feast-dev/feast/go/protos/feast/serving"
+	"github.com/feast-dev/feast/go/protos/feast/types"
+	"github.com/gocql/gocql"
+	"github.com/golang/protobuf/proto"
+	"github.com/rs/zerolog/log"
+
+	"google.golang.org/protobuf/types/known/timestamppb"
+	gocqltrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/gocql/gocql"
+)
+
+type CassandraOnlineStore struct {
+	project string
+
+	// Cluster configurations for Cassandra/ScyllaDB
+	clusterConfigs *gocql.ClusterConfig
+
+	// Session object that holds information about the connection to the cluster
+	session *gocql.Session
+
+	config *registry.RepoConfig
+}
+
+type CassandraConfig struct {
+	hosts                   []string
+	username                string
+	password                string
+	keyspace                string
+	protocolVersion         int
+	loadBalancingPolicy     gocql.HostSelectionPolicy
+	connectionTimeoutMillis int64
+	requestTimeoutMillis    int64
+}
+
+func extractCassandraConfig(onlineStoreConfig map[string]any) (*CassandraConfig, error) {
+	cassandraConfig := CassandraConfig{}
+
+	// parse hosts
+	cassandraHosts, ok := onlineStoreConfig["hosts"]
+	if !ok {
+		cassandraConfig.hosts = []string{"127.0.0.1"}
```

Comment:
Can you use constants wherever possible? 
Example: 
var DefaultCassandraHost = []string{"127.0.0.1"}

Wherever you're using hostnames, default values, etc.. Makes it easier to modify later instead of changeing it throughout the code.

Reply:
Would rather not, these values are only used in one place (`extractCassandraConfig(...)`) I think constants in this case make it harder to read without an IDE as you have to jump around.